pkg/grpc/portergrpc/context.go

```diff
@@ -0,0 +1,24 @@
+package portergrpc
+
+import (
+	"context"
+	"errors"
+
+	"get.porter.sh/porter/pkg/porter"
+)
+
+type ctxKey int
+
+const porterConnCtxKey ctxKey = 0
+
+func GetPorterConnectionFromContext(ctx context.Context) (*porter.Porter, error) {
```

Comment:
can you add a bit of godoc here? for example:

```suggestion
// Get the porter connection from the given context. Use AddPorterConnectionToContext to add one.
func GetPorterConnectionFromContext(ctx context.Context) (*porter.Porter, error) {
```